students/mattmaeda/mailroom_app/mailroom/cli.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+"""
+Interface to mailroom package
+"""
+
+from __future__ import print_function
+
+import sys
+import math
+import logging
+from mailroom import model, DATA_DIR
+
+logging.getLogger(__name__)
+
+MENU = """
+Choose an option:
+
+1. Send Thank You
+2. Create Report
+3. Quit
+
+> """
+
+DB = model.DonorDB.load_from_file(DATA_DIR / "sample_data.json")
+
+def get_selection():
+    """ Display menu option and get user input """
+    logging.info("Get menu selections")
+    selection = input(MENU)
+    logging.info("User selected option %s", selection)
+    return selection.strip()
+
+
+def send_thank_you():
+    """ Record a donation and send thank you message """
```

Comment:
I would consider breaking this into two functions, get_donor_name and get_donation_amount, or something like that. Your function would look something like this then:
```
def send_thank_you():
    donor = get_donor_name() # this function would run the while loop, add the donor to the database if it isn't there and return the name
    amount = get_donation_amount() # this function would run the second while loop, get the donation amount
    donor.add_donation(amount) 
    print(DB.send_letter(donor))
```